samples/bluetooth/central_otc/src/main.c

```diff
@@ -568,6 +588,10 @@ static int on_obj_data_read(struct bt_ots_client *ots_inst, struct bt_conn *conn
 		printk("Object total received %d\n", len + offset);
 		print_hex_number(obj_data_buf, len + offset);
 		(void)memset(obj_data_buf, 0, OBJ_MAX_SIZE);
+		otc_crc_work.offset = 0;
+		otc_crc_work.len = otc.cur_object.size.cur;
+		k_work_schedule(&otc_crc_work.work, K_MSEC(50));
```

Comment:
What if `&otc_crc_work.work` is already being used to calculate another checksum?

Reply:
oacp_checksum should return -EBUSY if there is already called.